website/docs/mlflow/mlflow_on_synapse_analytics.md

```diff
@@ -0,0 +1,161 @@
+# Log Model with MLflow on Synapse Analytics
```

Comment:
We have a few pages on MLFlow already, so we might be getting to the point where we should factorize and organize them. One potential factorization could be to make this page just cover installation of MLFlow on different environments. The next steps links could then go to the other two documents 

https://microsoft.github.io/SynapseML/docs/mlflow/examples/
https://microsoft.github.io/SynapseML/docs/mlflow/autologging/

Those documents should contain pointers back to this document in the prerequisites. Likewise, the overview doc

https://microsoft.github.io/SynapseML/docs/mlflow/introduction/

Should point to this installation doc to guide users through. Also one thing that might make this simpler too is to combine the demo in this doc, with the other two markdowns on example usage and autologging and make just a single usage document that gives people a few minimal examples, and goes over how to configure autologging